app/src/main/java/nl/avans/drivioapp/UpdateCarFragment.kt

```diff
@@ -0,0 +1,99 @@
+package nl.avans.drivioapp
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.setFragmentResultListener
+import androidx.fragment.app.viewModels
+import nl.avans.drivioapp.databinding.FragmentAddElectricCarBinding
+import nl.avans.drivioapp.model.ElectricCar
+import nl.avans.drivioapp.model.UserElectricCar
+import nl.avans.drivioapp.viewModel.AddElectricCarViewModel
+import nl.avans.drivioapp.viewModel.MyCarsViewModel
+
+class UpdateCarFragment : Fragment(R.layout.fragment_add_electric_car) {
+    private val addElectricCarViewModel: AddElectricCarViewModel by viewModels();
+    private var _binding: FragmentAddElectricCarBinding? = null;
+    private val binding get() = _binding!!;
+    private val myCarsViewModel: MyCarsViewModel by viewModels()
+    private var carId: Int? = null;

```

Comment:
Deze kan eruit aangezien er een local variable aanwezig is.

Reply:
Deze wordt later ook weer gebruikt (line 72), vandaar dat ik het op deze manier heb gedaan

Reply:
Ah, check. Die had ik niet gezien.